app/src/main/kotlin/com/simplemobiletools/dialer/App.kt

```diff
@@ -1,11 +1,19 @@
 package com.simplemobiletools.dialer
 
 import android.app.Application
+import android.content.Context
 import com.simplemobiletools.commons.extensions.checkUseEnglish
 
 class App : Application() {
     override fun onCreate() {
         super.onCreate()
+        context = this;
         checkUseEnglish()
     }
+
+
+    companion object {
+        lateinit var context: Context
```

Comment:
definitely not a good idea

Reply:
How would you access the string resources otherwise?

Reply:
by passing the local context wherever needed